ethereum/contracts/DOTApp.sol

```diff
@@ -118,7 +122,7 @@ contract DOTApp is FeeController, AccessControl {
         // revoke old channel
         revokeRole(INBOUND_CHANNEL_ROLE, c1.inbound);
         revokeRole(INBOUND_CHANNEL_ROLE, c2.inbound);
-        // set new channel
+       // set new channel
```

Comment:
Nit: whitespace

Reply:
Thanks, though this code is all going to be removed by my other open prs.